android-interop-testing/app/proguard-rules.pro

```diff
@@ -18,6 +18,8 @@
 -dontwarn sun.reflect.**
 # Ignores: can't find referenced class javax.lang.model.element.Modifier
 -dontwarn com.google.errorprone.annotations.**
+-dontwarn org.apache.commons.logging.**
```

Comment:
where is apache logging from? add a comment?

Reply:
No longer necessary, reverted